app/collections/show.cjsx

```diff
@@ -34,27 +38,61 @@ CollectionPage = React.createClass
   componentWillUnmount: ->
     document.documentElement.classList.remove 'on-collection-page'
 
+  getBaseLink: ->
+    baseLink = "/"
+    if @props.project?
+      baseLink += "projects/#{@props.project.slug}"
+    baseLink
+
+  getOwnerLogin: ->
+    [owner_login, collection_name] = @props.collection.slug.split('/')
+    owner_login
+
+  getCollectionsLink: ->
+    baseLink = @getBaseLink()
+    [owner_login, collection_name] = @props.collection.slug.split('/')
+    if @props.collection.favorite
+      "#{baseLink}favorites/#{@getOwnerLogin()}"
```

Comment:
happy to change these but out of interest why does it matter if the slash comes from the baseLink function/variable or from the string itself?